src/types/wiki.ts

```diff
@@ -58,6 +58,7 @@ export enum CommonMetaIds {
   // other info
   WEBSITE = 'website',
   CONTRACT_URL = 'contract_url',
+  EVENT_LOCATION = 'event_location',
```

Comment:
call it location so its more generic and can be used in other wikis

Reply:
done